database/migrations/2023_10_04_172124_add_role_id_to_users_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use App\Models\Role;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->foreignIdFor(Role::class)->default(RoleName::MEMBER)->constrained()->cascadeOnDelete();
```

Comment:
Same as above, we cannot be sure if the number will always be the same for every environment so we can't really reference it like this.

Now when I think about it, you can populate the column by setting it to nullable first, then loop over the users and assign them as members and remove the nullable from the role_id column.